tracee-rules/signature_test.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 func Test_getSignatures(t *testing.T) {
-	sigs, err := getSignatures("signatures/rego", []string{"TRC-2"})
+	sigs, err := getSignatures("./dist/rules", []string{"TRC-2"})
```

Comment:
Passing `signatures/rego` makes the use of existing rego signatures as inputs, effectively using them as golden files for this test.

Was there another motivation to change it to `dist/rules`?

Reply:
I think it's ok to leave the helpers under the signatures directory, after all they are signature helpers. that will also address this concern I believe